### DISCUSSION TITLE
Add microwatt to list of predefined units?

### DISCUSSION THREAD

**Discussion by simonvanderveldt** [grafana/grafana#38121] [General]
Would it be possible to add microwatt to the list of predefined units? This is for example used for power usage reporting in  (certain parts of) the Linux kernel.

It seems like it wouldn't be more than adding something like
```
{ name: 'Microwatt (µW)', id: 'µwatt', fn: decimalSIPrefix('W', -2) },
```
here https://github.com/grafana/grafana/blob/3039f9c3bd2796a0a410544ad2e0e1ed4426a8db/packages/grafana-data/src/valueFormats/categories.ts#L183

I don't really know how to make/test changes to Grafana though, so haven't been able to verify this.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
